app/src/main/kotlin/com/wire/android/mapper/MessagePreviewContentMapper.kt

```diff
@@ -82,34 +82,80 @@ fun MessagePreview.uiLastMessageContent(): UILastMessageContent {
                     userUIText,
                     UIText.StringResource(R.string.last_message_knock)
                 )
-                is WithUser.MembersAdded -> UILastMessageContent.SenderWithMessage(
-                    userUIText,
-                    UIText.PluralResource(
-                        R.plurals.last_message_people_added,
-                        (content as WithUser.MembersAdded).userIdList.size,
-                        (content as WithUser.MembersAdded).userIdList.size,
-                    )
-                )
+                is WithUser.MembersAdded -> {
+                    val membersAddedContent = (content as WithUser.MembersAdded)
+                    val isSelfAdded = membersAddedContent.isSelfUserAdded
+                    val isSelfJoined = isSelfMessage && isSelfAdded
+
+                    when {
+                        isSelfJoined -> UILastMessageContent.TextMessage(
+                            MessageBody(UIText.StringResource(R.string.left_conversation_group_success))
```

Comment:
For `MembersAdded` there should "**joined**_conversation_group_success", right? 😄